Formula/fastq-tools.rb

```diff
@@ -0,0 +1,34 @@
+class FastqTools < Formula
+  desc "Small utilities for working with fastq sequence files"
+  homepage "https://github.com/dcjones/fastq-tools"
+  url "https://github.com/dcjones/fastq-tools/archive/refs/tags/v0.8.3.tar.gz"
+  sha256 "0cd7436e81129090e707f69695682df80623b06448d95df483e572c61ddf538e"
+  license "MIT"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
```

Comment:
Do we need this? The upstream doesn't use latest release feature and there doesn't seem to be any pre-releases in tags https://github.com/dcjones/fastq-tools/releases

The default livecheck seems like it would work here.

Reply:
Oh interesting, I wasn't aware of the issues around github_latest. I'll push a fix.

Reply:
It isn't really an issue as livecheck works, but I think we prefer `git` over `github_latest` due to rate-limiting.